### PR TITLE
[KHWEBA-35] first configuration

### PR DESCRIPTION
**Pull request by gift-a**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "buxy-ng",
+  "version": "0.0.1",
```
Comment (ihor7016) [inline package.json]: version is `2.0.0`

File: package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "description": "Simple finance tracking app",
+  "scripts": {
+    "start": "webpack-dev-server  --mode=development",
```
Comment (ihor7016) [inline package.json]: why 2 spaces here?

**Comment by ihor7016** [comment]
where is editorconfig?

**Comment by gift-a** [comment]
it is in master

**Review by ihor7016 — APPROVED** [review]

**Pull request merged**